ai/smart-labeling/how-to/access-annotation-server.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: How to access the annotation server
+description: This page explains how to access the annotation server
+dates:
+  validation: 2021-11-05
+  posted: 2021-11-05
+---
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
+- You have a [Smart Labeling](/ai/smart-labeling/quickstart) workspace
+
+</Message>
+
+1. Click **Smart Labeling** in the **AI** section of the side menu. The list of your workspaces displays.
+
+2. Click on the workspace you want to access. The workspace overview displays.
+
+3. Click **Go to my annotation server** in the workspace overview:
+
+<Lightbox src="scaleway-smart-labeling-worksapce.png" />
+
+The CVAT login screen dispalys.
+
+<Message type="tip">
+
+Alternatively you can use the **Annotation server URL** to access the server. You can find it in the workspace information block. This URL will take you (or the annotator) to the CVAT annotation interface. 
+
+</Message>
+
+4. Enter the credentials for the admin user of the Smart Labeling workspace and click **Login**. The CVAT dashboard displays:
+
+<Lightbox src="scaleway-smart-labeling-dashboard.png" />
+
+5. Ceate Tasks to annotate datasets of your choice with [image-level tags, as well as points, bounding boxes, polygons, and 3D boxes](https://openvinotoolkit.github.io/cvat/docs/manual/basics/types-of-shapes/) and assign them to your users.
+
+6. Users can login into the CVAT interface to start annotating images in their tasks.
```

Comment:
I think this should not be a numbered step, since not an action verb. Or else maybe a tip?